_posts/2021-03-15-cloud-migration.md

```diff
@@ -0,0 +1,86 @@
+I've summarized our discussion below,
+and hope it helps you understand the hows and whys of the cloud migration project.
+
+**How did you decide how to set up the new AWS cloud system?**
+
+We talked to a subset of existing cloud users on campus, 
+specifically asking about pain points, such as:
+what tasks in the cloud are difficult to accomplish?
+Moreover,
+we looked at how the existing infrastructure had developed over the years.
+Reverse engineering the process allowed us to see why specific features had been designed in certain ways,
+and helped us anticipate some of the difficult parts of implementing the new system.
+
+**How does the new infrastructure help reproducibility?**
+
+SciComp dedicates a lot of time and energy to supporting reproducible computational methods.
+The new cloud environment will enable reproducibility in ways that are especially important for our computational community.
+The new infrastructure allows retention and sharing of standardized compute environments,
+such that previously applied workflows can be easily used to rerun the analysis on new samples,
+or can even be shared with other labs.
+Moreover, 
+this means all existing (and any newly emerging)
+services for genomics and biomedical research available from AWS can be deployed with minimal configuration on the part of researchers.
+
+**What do you wish you'd known at the beginning about supporting researchers in the cloud data and compute?**
+
+Many cloud users at Fred Hutch are only interested in data storage,
+so we often work with labs and projects to move data to the cloud,
+or help them receive data from collaborators off campus using cloud resources.
+The nature and quantity of data being generated by many labs means that organizing research data continues to be really difficult.
+Additionally, the way that researchers use cloud resources differs widely among groups.
+Nearly every group has a specific need or use for cloud infrastructure that differs from all other groups,
+and we would like to be able to support everyone's needs.
+This means we're continuing to build functionality into our infrastructure as we migrate labs to the new system.
+The migration process, therefore, hasn't been as easy a transition as we would like,
+but we're proud to be able to meet these needs and continue facilitating cutting-edge science.
+
+**What do you find the most exciting about the new AWS infrastructure?**
+
+From a technical perspective,
+we're very pleased with the ability provided by the new AWS infrastructure to make updates that can be applied to all users within minutes. 
+This makes it easy for us to open up new opportunities for everyone across campus.
+
+We've also been careful about the methods we use to grant and manage access to cloud resources.
+Collaborators external to the Hutch now have the ability to access Hutch accounts with their own credentials,
```

Comment:
External collaborators have always had their own credentials but we typically would provide them.  The difference is that if the collaborator's institution has an AWS account, we can now use the credentials from the collaborator's institution without the need for temporary users or additional credentials.  Enabling this type of access is much more secure and the setup is both faster and easier to do.